ImageGoNord/__init__.py

```diff
@@ -1,4 +1,11 @@
 # gonord version
 __version__ = "1.2.0"
```

Comment:
We should use only one `__version__` tag and update this at lease with as we introduce here a client

```suggestion
__version__ = "1.3.0"
```

Reply:
I think we could change this to dinamically take tags